BrightCoders-Institute/BCDIC22-RoR-reloj-checador-team2
# Database connection employees register form

# Description

What did you implement/Why did you implement this:

- Employee register form created
- Database connection between register employee form and employee table

## Test

* Move to /employees/new path or click in the add employee button in the /employee path
* Fill the form and submit
* Check the employee's table for the employee created

## Screenshots

![image](https://user-images.githubusercontent.com/84949558/214695253-8c02d1d8-3b79-41f3-8bc4-91e06f0dd735.png)
